code/modules/religion/rites.dm

```diff
@@ -603,4 +603,101 @@
 	new /obj/item/ritual_totem(altar_turf)
 	return TRUE
 
+///sparring god rites
 
+/datum/religion_rites/sparring_contract
+	name = "Summon Sparring Contract"
+	desc = "Turns some paper into a sparring contract."
```

Comment:
reducing the time it takes to complete this ritual would be nice, it takes forever to amass a decent pile of paperwork for if you wanna host a bunch of duels in a row

Reply:
I actually went in the opposite direction here! There's now only one contract at a time in existence.